templates/rate-plan-detail.html.twig

```diff
@@ -6,7 +6,12 @@
  * Variables:
  * - detail
  *   - id:
- *   - ratePlanRates:               Rate plan rate details, such as the type of rate plan (REVSHARE or RATECARD), the rate for a rate card plan, the revenue share for a revenue share plan, and the range (starting unit and ending unit for which the rate plan rate applies).
+ *   - ratePlanRates:               All rate plan rate details, such as the type of rate plan (REVSHARE or RATECARD), the rate for a rate card plan, the revenue share for a revenue share plan, and the range (starting unit and ending unit for which the rate plan rate applies).
+ *     - id:                          The unique ID of the rate plan rate.
+ *     - startUnit:                   The number the rate starts at.
+ *     - endUnit:                     The rate is good until this nuber of units.
+ *     - rate:                        The rate that is applies within this range.
+ *     - revshare:                    The revenue share percentage.
  *   - aggregateFreemiumCounters:   Flag that specifies whether or not aggregate counters are enabled to determine if usage of an API product is in the free range. Aggregate counters must be enabled to set up a freemium plan for a product.
  *   - aggregateStandardCounters:   Flag that specifies whether or not aggregate counters are used to determine the band of usage (such as a volume band for a rate card plan).
  *   - aggregateTransactions:
```

Comment:
There are no docs for this line either?  Maybe it's not needed?

Reply:
Yeah, it's not being used by monetization so I removed it.